src/core/shards/core.hpp

```diff
@@ -1192,9 +1192,15 @@ struct Swap {
     _targetB = referenceVariable(ctx, _nameB.c_str());
   }
 
+  static SHOptionalString help() {
+    return SHCCSTR("Swaps the values of the two variables passed to it via `:NameA` and `:NameB` parameters.");
+  }
+
   static SHTypesInfo inputTypes() { return CoreInfo::AnyType; }
+  static SHOptionalString inputHelp() { return SHCCSTR("NA"); }
```

Comment:
If it means "non applicable", the usual abbreviation is `"N/A"`.

See https://en.wikipedia.org/wiki/N/A